systemtests/src/main/java/com/github/streamshub/systemtests/setup/strimzi/KafkaSetup.java

```diff
@@ -147,6 +170,7 @@ public static Kafka getDefaultKafka(String namespaceName, String clusterName, St
                     .addToConfig("transaction.state.log.replication.factor", Math.min(replicas, 3))
                     .addToConfig("default.replication.factor", Math.min(replicas, 3))
                     .addToConfig("min.insync.replicas", Math.min(Math.max(replicas - 1, 1), 2))
+                    .addToConfig("allow.everyone.if.no.acl.found", "true")
```

Comment:
If you configured ACLs, this can be removed?

Reply:
yes, good catch